shell/src/test/java/alluxio/cli/fsadmin/report/CapacityCommandTest.java

```diff
@@ -153,6 +154,19 @@ public void longWorkerNameCapacity() throws IOException {
     }
   }
 
+  /**
+   * Check whether WorkerInfoField class and WorkerInfoField in proto file has identical fields.
+   */
+  @Test
+  public void workerInfoFieldCapacity() throws IOException {
+    // If Options has a WorkerInfoField which POptions does not have, throw IOException.
+    GetWorkerReportOptions.defaults().toProto();
```

Comment:
for a test to work, you need to encode the property being tested as an assertion, e.g.

```java
assertEquals(GetWorkerReportPOptions.getDefaultInstance(),
    GetWorkerReportOptions.defaults().toProto());
```

Ideally, you don't need to waste time writing comments explaining what is being tested, but rather the test code should be self-explanatory as if you are reading a plain English sentence. In this case, you can read it aloud as "the default value of a gRPC `GetWorkerReportOptions` equals the default value of the java instance returned by the `default` method and then converted to the gRPC representation."